chrome-extension/src/App.js

```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react';
+import './App.css';
+import {v4 as uuid} from "uuid";
+
+function App() {  
+
+  const [userName,setUserName]=useState()
+  const [userEmail,setUserEmail]=useState("")
+  const [isEmail,setIsEmail]=useState(false)
+  const [showName,setShowName]=useState(true)
+  const [isPassword,setIsPassword]=useState(false)
+  const [Time,setTime]=useState()
+  const [focus,setFocus]=useState(false)
+  const [greeting,setGreeting]=useState()
```

Comment:
you can use reducer over here since states are quite more in number

Reply:
yes will do it in final version.